### ISSUE TITLE
Reddit button links to broken page

### ISSUE THREAD

**Issue by ronnyli** [ronnyli/thredd#114]
Just need to remove the `//` (double slashes) with a single slash `/`

See u/iVarun's comment:
https://www.reddit.com/r/Thredd/comments/fp43a4/coming_soon_options_for_turning_off_thredd_next/fo3ap4s/?context=3

**Comment by mmortal03** [comment]
Yes, this makes this extension unusable with RES when you browse to the direct link. Please fix this.
